src/lib/nextWordGuesserStrategy.ts

```diff
@@ -79,10 +107,41 @@ export abstract class NextWordGuesserStrategyBase implements INextWordGuesserStr
         return maxBy(wordsWithGuesses, 'score') as WordGuessAndScore;
     }
 
+    /**
+     * @returns the list of all unique letters already guessed, regardless of position.
+     */
     getAlreadyGuessedLetters(): string[] {
         return uniq(this.previousGuesses.flatMap((guess) => Array.from(guess.word)));
     }
 
+    /**
+     * @returns true if the given `WordList` is solved (has only one possible word); false otherwise.
+     *
+     * @example A single-word `WordList` is trivially already solved:
+     * ```typescript
+     * const abcWordList = new WordList(['ABC']);
+     * const guesserStrategy = new NextWordGuesserStrategyBaseTest(abcWordList);
+     * expect(guesserStrategy.isSolved()).toBe(true);
+     * ```
+     */
+    isSolved(): boolean {
+        return this.wordList.words.length === 1;
+    }
+
+    /**
+     * @returns true if the given WordList is solvable (has at least one possible word); false otherwise.
+     *
+     * @example An empty `WordList` is trivially not solvable:
+     * ```typescript
+     * const emptyWordList = new WordList([]);
+     * const guesserStrategy = new NextWordGuesserStrategyBaseTest(abcWordList);
```

Comment:
Fix copy-pasta'ed example.
```suggestion
     * const guesserStrategy = new NextWordGuesserStrategyBaseTest(emptyWordList);
```